src/ci/docker/scripts/android-ndk.sh

```diff
@@ -4,28 +4,10 @@ set -ex
 URL=https://dl.google.com/android/repository
 
 download_ndk() {
-    mkdir -p /android/ndk
-    cd /android/ndk
+    mkdir /android/
+    cd /android
     curl -fO $URL/$1
     unzip -q $1
     rm $1
     mv android-ndk-* ndk
 }
-
-make_standalone_toolchain() {
-    # See https://developer.android.com/ndk/guides/standalone_toolchain.htm
```

Comment:
looks like this is all no longer necessary in new releases :) cool! https://developer.android.com/ndk/guides/standalone_toolchain.html

Reply:
Yeah, the NDK team has really put a lot of work into making it usable right out of the box.